packages/bpk-component-banner-alert/src/BpkBannerAlert.module.scss

```diff
@@ -124,14 +124,6 @@
       fill: $bpk-text-secondary-day;
     }
 
-    &__toggle-button {
-      padding: 0;
-      border: 0;
-      background-color: transparent;
-      cursor: pointer;
-      appearance: none;
-    }
-
```

Comment:
If we remove this it causes this issue with the toggle button to occur 

![Screenshot 2024-01-08 at 09 44 31](https://github.com/Skyscanner/backpack/assets/8831547/ce2c63e4-71b8-4b48-a090-aac744702bac)

https://backpack.github.io/storybook-prs/3148/?path=/story/bpk-component-banner-alert--success-expandable-behaviour